packages/dataviews/src/dataform-controls/index.tsx

```diff
@@ -29,6 +30,7 @@ const FORM_CONTROLS: FormControls = {
 	radio,
 	select,
 	text,
+	'toggle-group': toggleGroup,
```

Comment:
We tend to prefer camelCase to kebab-case in these things. Any reason for using kebab-case here?

Reply:
No reason; just an arbitrary choice. Will update!